modules/standard/LinkedLists.chpl

```diff
@@ -193,28 +193,80 @@ record LinkedList {
      if boundsChecking && size < 1 {
        HaltWrappers.boundsCheckHalt("pop_front on empty list");
      }
-     var oldfirst = first!;
-     var newfirst = first!.next;
+     var oldfirst = _first!;
+     var newfirst = _first!.next;
      var ret = oldfirst.data;
-     first = newfirst;
-     if last == oldfirst then last = newfirst;
+     _first = newfirst;
+     if _last == oldfirst then _last = newfirst;
      size -= 1;
      delete oldfirst;
      return ret;
    }
 
+  /*
+    find `e` in the list.
+
+    :arg e: The element to be checked for
+    :return: True if the item was found, otherwise returns false
+    :rtyle: `bool`
+   */
+  proc contains(const e: eltType): bool {
+    for item in this {
+      if (e == item) {
+        return true;
+      }
+    }
+    return false;
+  }
+  
+  /*
+    Returns a reference to the first item in the list
+    
+    .. warning::
+      Calling this method on an empty list will cause the program to halt
```

Comment:
How about "Calling this method on an empty list will cause the currently running program to halt. If the `--fast` flag is used, no safety checks will be performed."